Resources/input/fsh/ISiKDataTypeProfiles.fsh

```diff
@@ -0,0 +1,72 @@
+Profile: ISiKCoding
+Parent: Coding
+Id: ISiKCoding
+Title: "ISiKCoding"
+Description: "Data Type profile for Codings in ISiK"
+* system 1.. MS
+* code 1.. MS
+* display MS
```

Comment:
question: geben wir nicht selber häufiger KEIN display an , z.B. bei pattern?

Reply:
das ist korrekt, einen bestimmten display value sollte man nicht in Profilen erzwingen, in der Instanz sollten aber display values vorhanden sein (falls bekannt, daher MS)